corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -976,16 +976,24 @@ def get_case_ids_modified_with_owner_since(domain, owner_id, reference_date):
             return [result.case_id for result in results]
 
     @staticmethod
-    def get_extension_case_ids(domain, case_ids):
+    def get_extension_case_ids(domain, case_ids, include_closed=True):
         """
         Given a base list of case ids, get all ids of all extension cases that reference them
         """
         if not case_ids:
             return []
-        with get_cursor(CommCareCaseIndexSQL) as cursor:
-            cursor.execute('SELECT case_id FROM get_extension_case_ids(%s, %s)', [domain, list(case_ids)])
-            results = fetchall_as_namedtuple(cursor)
-            return [result.case_id for result in results]
```

Comment:
I'll remove the SQL functions in a separate PR which can be deployed after this code is deployed (to avoid 'missing sql function' errors during deploy.